examples/example-app-router/src/i18n.ts

```diff
@@ -1,5 +1,10 @@
 import {getRequestConfig} from 'next-intl/server';
 
 export default getRequestConfig(async ({locale}) => ({
-  messages: (await import(`../messages/${locale}.json`)).default
+  messages: (
+    await (locale === 'en'
+      ? // When using Turbopack, this will enable HMR for `en`
+        import(`../messages/en.json`)
```

Comment:
Quite handy!

Reply:
Indeed! Is there a way we could iterate through the available languages (all available languages and load them directly here?)

Is that even necessary? Would loading all languages without string interpolation be problematic?

Because by that snippet you're still loading conditionally, but how we could do "magic" and make Turbopack understand that it can load all the languages in that fashion?

Reply:
> Would loading all languages without string interpolation be problematic?

That should work too, yes! Depending on how many locales your app has (and how often they change), this could surely be a good idea!

Reply:
I see thar HMR benefit is mostly just local dev and development is always in English, so just for HMR that might not be necessary.

What I wonder if not using strin interpolation here brings any other Turbopack magic benefit

Reply:
> I see that HMR benefit is mostly just local dev and development is always in English, so just for HMR that might not be necessary.

That's what I thought too, yes!